platform/pulp/plugin/changeset/report.py

```diff
@@ -0,0 +1,69 @@
+class Action:
+    ADDED = 'ADDED'
+    DELETED = 'DELETED'
+
+
+class Disposition:
```

Comment:
Since this never gets used as an object, I think it's here just as a namespace. Is there another way to communicate the grouping without using a class perhaps?